argobots.spec

```
@@ -1,22 +1,27 @@
 Name: argobots
-Version: 1.0
-Release: 1%{?dist}
+Version: 1.1a1
```

```suggestion
Version: 1.1~a1
```

Pre-release tags in versions should be separated from the main version with a `~` so that RPM can properly compare versions.  I.e. `1.1~a1 < 1.1` however `1.1a1 > 1.1`, so without the `~`, a `1.1` versioned package will not upgrade a `1.1a1` versioned package, which of course it should.

---

ok